src/partials/about.html

```diff
@@ -0,0 +1,21 @@
+<section id="how its made" class="container">
+  <p class="about">TRADICIONAL AND LOVE</p>
+
+  <p class="about__title1">HOW IT`S MADE ?</p>
```

Comment:
Użyj tutaj tagu <h2>, jest to nagłówek sekcji

Reply:
chodzi o id? w wytycznych jest "Przyciski pod tytułem “Products” i “How it’s made” to linki kotwiczne do odpowiednich sekcji." myslalem ze aby rzycisk pzrewijal strone do miejsca do ktorego ma przwinac to w ktoryms elemencie ma byc id="xxxxx"

Reply:
Cały komentarz się źle napisał, chodziło mi o to, że nagłówek sekcji musi być w tagu h2 zamiast p